sql/sql.y

```diff
@@ -166,12 +173,16 @@ select
 ;
 
 train_clause
-: TRAIN IDENT WITH attrs COLUMN columns LABEL IDENT INTO IDENT {
+: TRAIN IDENT WITH attrs {
 	$$.estimator = $2
 	$$.attrs = $4
-	$$.columns = $6
-	$$.label = $8
-	$$.save = $10
+  }
+| train_clause COLUMN columns_clause {
+	$$.ccs = append($$.ccs, $3)
+  }
+| train_clause LABEL IDENT INTO IDENT {
+	$$.label = $3
+	$$.save = $5
```

Comment:
There is a problem with this change, as it makes our parser accepts the following writings


```
TRAIN DNNClassifiere LABEL a INTO model1
TRAIN DNNClassifiere LABEL a INTO model1 LABEL b INTO model2
TRAIN DNNClassifiere LABEL a INTO model1 LABEL b INTO model2 LABEL c INTO model3
```

However, we want only one LABEL clause and one INTO clause.

My understanding is that we may not change too much of the rule train; instead, we might want just a little change from `COLUMN columns` into `column_clause`:

```
train : TRAIN IDENT column_clause LABEL IDENT INTO IDENT
```

where 

```
column_clause
: COLUMN exprlist
: column_groups

column_groups
: COLUMN exprlist FOR IDENT
| column_groups COLUMN exprlist FOR IDENT
````

Please be aware that the above rules don't accept

```
COLUMN a, b COLUMN c, d
```

which is what we want.

Reply:
Thanks for pointing that out, it's indeed a problem, will fix it.

Reply:
As your suggestion, I have recovered the rule of `train_clause` and do the change only in `column_clause`.

Because I don't want to do the restriction that the location of `COLUMN ... FOR` must be after the anonymous `COLUMN` or there must be only one anonymous `COLUMN`, the rule of `column_clause` looks like this:
`column_clause
: COLUMN columns
| COLUMN columns FOR IDENT
| column_clause COLUMN columns
| column_clause COLUMN columns FOR IDENT
`

If there is more than one anonymous `COLUMN` or `COLUMN`s which have the same `FOR IDENT`, the latter one makes effects.

@wangkuiyi What do you think?